x-pack/legacy/plugins/siem/public/containers/query_template_paginated.tsx

```diff
@@ -31,15 +35,24 @@ export class QueryTemplatePaginated<
   TData = any,
   TVariables = OperationVariables
 > extends React.PureComponent<T, TData, TVariables> {
+  private queryVariables: TVariables | null = null;
+  private myLoading: boolean = false;
   private fetchMore!: (
     fetchMoreOptions: FetchMoreOptionsArgs<TData, TVariables>
   ) => PromiseApolloQueryResult;
 
   private fetchMoreOptions!: (newActivePage: number) => FetchMoreOptionsArgs<TData, TVariables>;
 
+  public memoizedRefetchQuery: (
+    variables: TVariables,
+    limit: number,
+    refetch: (variables?: TVariables) => Promise<ApolloQueryResult<TData>>
+  ) => inputsModel.Refetch;
+
   // eslint-disable-next-line @typescript-eslint/no-useless-constructor
```

Comment:
This linter rule was added around 2 months ago by core team. It looks like we can remove it now since this is no longer a useless constructor.